### ISSUE TITLE
abnormality of Schwann cells

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#5791]
**Preferred term label:**

abnormality of Schwann cells

**Synonyms**

abnormality of Neurilemmal cells; abnormality of neurilemmocytes

**Definition (free text, please give PubMed ID)**

Abormality of schwann cells, glial cells that ensheath axons of neurons in the peripheral nervous system and are necessary for their maintenance and function.
CL:0002573; pmid: 31760652

**Parent term (use hpo.jax.org/app)**

Abnormality of the glial cells (HP:0100705)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

demyelinating disease (DOID:3213)

**Your nano-attribution (ORCID)**

ORCID: 0000-0002-9553-7227

**Comment by pnrobinson** [comment]
added as HP:0033035
